server/graphql/core/src/loader/name.rs

```diff
@@ -62,3 +62,62 @@ impl Loader<NameByIdLoaderInput> for NameByIdLoader {
         Ok(output)
     }
 }
+
+#[derive(Clone)]
+pub struct ByNameLinkEmptyPayload;
+pub type NameByNameLinkIdLoaderInput = IdPair<ByNameLinkEmptyPayload>;
+
+impl NameByNameLinkIdLoaderInput {
```

Comment:
👀  All the NameById loader calls should be using this~ new issue? 👀

Reply:
not necessarily? Lots of places, we have like `Invoice`, which has `name_row` (already went via link table in repository, we just use loader to get name's related records like store_row for the full NameNode?